semantics/language/common/expr/reference.k

```diff
@@ -67,6 +67,7 @@ module C-COMMON-EXPR-REFERENCE
      }*/
      rule *(tv(Loc:SymLoc, T:PointerType))
           => checkRestrict(Loc)
+          ~> checkLoc(Loc)
```

Comment:
It seems to happen only if the address is of static duration. Do you know why that might be?